### DISCUSSION TITLE
Cache restore issue in GITHUB Action

### DISCUSSION THREAD

**Discussion by pratik7673** [community/community#125157] [Actions]
### Body

Hello All,

Recently we are seeing some issue in cache artifact, actions/cache@v3 is not working as expected. anyone also facing the same issue
Cache not found for input keys: 5117c2268c09d9cff6c2f64c0b9c7012f9ea4b4f-172430

**Comment by zhztheplayer** [comment]
The same. In our project cache created in base branch can no longer be restored from PR's forked branches.
